gateway/engine/core/src/main/java/io/apiman/gateway/engine/impl/DefaultPluginRegistry.java

```diff
@@ -104,11 +105,7 @@ public DefaultPluginRegistry(File pluginsDir, Set<URI> pluginRepositories) {
     private static final File createTempPluginsDir() {
         // TODO log a warning here
         try {
-            @SuppressWarnings("nls")
-            File tempDir = File.createTempFile("_apiman", "plugins");
-            tempDir.delete();
```

Comment:
I seem to recall this was some sort of hack for Windows xx @EricWittmann ?

Reply:
I don't recall that.

Reply:
I think this change is OK.